examples/hosted-delivery-function-potterdb/delivery-functions/index.ts

```diff
@@ -37,11 +43,11 @@ const queryHandler: EventHandler<EventType.GRAPHQL_QUERY> = async (event, contex
   return response.json()
 }
 
-export const handler: EventHandler = (event, context) => {
-  if (event.type === EventType.GRAPHQL_FIELD_MAPPING) {
+export const handler: EventHandler<EventType> = (event, context) => {
```

Comment:
The generic argument should default to all types automatcially
```suggestion
export const handler: EventHandler = (event, context) => {
```